components/form/Textarea.js

```diff
@@ -3,45 +3,107 @@ import { useFormContext } from 'react-hook-form';
 import PropTypes from 'prop-types';
 import { Label, ErrorMessage } from '.';
 import classNames from 'classnames';
+import { HelpIcon, HintIcon, SuccessIcon } from '@/public/assets/dashboard/navBarIcon';
 
-const Textarea = ({ className, label, labelClassName, name, placeholder }) => {
+const Textarea = ({
+  className,
+  floatLabel,
+  label,
+  leadingIcon,
+  labelClassName,
+  name,
+  placeholder,
+  rows,
+  withHintIcon,
+  withHintText
```

Comment:
No sure why you added `with` to the variable names. I believe we could use a better name based on the usecase.